x-pack/plugins/core_telemetry/server/lib/collectors/visualizations/register_usage_collector.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IHapiServer } from '../../../../';
+import { getUsageCollector } from './get_usage_collector';
+
+export function getVisualizationsCollector(server: IHapiServer): void {
+  const { usage } = server;
+  const collector = usage.collectorSet.makeUsageCollector(getUsageCollector(server));
+  server.usage.collectorSet.register(collector);
```

Comment:
This could just be `usage.collectorSet.register(collector);`, as you've destructured `usage` on line 11.

Reply:
d648e7926c